src/aurora.provision-database.ts

```diff
@@ -191,28 +199,47 @@ export async function createUpdate(props: CreateUpdateProps): Promise<awsLambda.
   const m = new Methods();
 
   let clientConfig: ClientConfig;
-  try {
-    console.log('Fetching credentials from Secrets Manager');
-    clientConfig = await m.fetchSecret(managerSecretArn);
-  } catch (err) {
-    return resultFactory({
-      ReasonPrefix: `Secrets issue: ${err}`,
-      Status: CfnStatus.FAILED,
-    });
-  }
-
   let client: Client;
-  try {
-    console.log(`Connecting to database "postgres"`);
-    // While the grants below care which database we are in,
-    // we haven't created the database yet, so we need to connect to postgres.
-    client = await m.connect({ ...clientConfig, database: 'postgres' });
-  } catch (err) {
-    console.log(`connect failed: ${err}`);
-    return resultFactory({
-      ReasonPrefix: `connect failed: ${err}`,
-      Status: CfnStatus.FAILED,
-    });
+
+  let passwordAuthenticationFailedRetries = maxRetries;
```

Comment:
Just naming stuff:
```suggestion
  let passwordAuthenticationLeftRetries = maxRetries;
```

Reply:
or `passwordAuthenticationFailedRetries = 0` - this would require changing the condition inside the loop